### ISSUE TITLE
[android_intent] Plugin should support sending broadcast intents

### ISSUE THREAD

**Issue by Max-Might** [flutter/flutter#65533]
## Use case

Sending broadcast intents.

## Proposal
1. Construct an intent.
```dart
final intent = const AndroidIntent(action: 'android.intent.action.CLOSE_SYSTEM_DIALOGS');
```
2. Send the intent as a broadcast with `intent.sendBroadcast()` instead of `intent.launch()`.

This is equivalent to calling the following native code
```java
Intent closeIntent = new Intent(Intent.ACTION_CLOSE_SYSTEM_DIALOGS);
context.sendBroadcast(closeIntent);
```

**Comment by miquelbeltran** [comment]
FYI, this has been implemented in the android_intent_plus package v0.4.2

**Comment by stuartmorgan** [comment]
Closing as `android_intent` is deprecated in favor of [`android_intent_plus`](https://pub.dev/packages/android_intent_plus).
